test/util/nightwatch-commands/fillAddress.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Fills the address form elements.
+ */
+exports.command = function fillAddress(schemaformBaseName, address) {
```

Comment:
Yeah, I originally had it as `baseName`, but I wanted it to be obvious that it was supposed to be `root_parentField_childField`. If it's still obvious, I can shorten it back down to `baseName`.